cmd/copyEnumeratorInit.go

```diff
@@ -361,6 +374,41 @@ func (cca *cookedCopyCmdArgs) createDstContainer(containerName, dstWithSAS strin
 		} else {
 			return err
 		}
+	case common.ELocation.File():
+		// Grab the account root and parse it as a URL
+		accountRoot, err := GetAccountRoot(dstWithSAS, cca.fromTo.To())
+
+		if err != nil {
+			return err
+		}
+
+		dstURL, err := url.Parse(accountRoot)
+
+		if err != nil {
+			return err
+		}
+
+		fsu := azfile.NewServiceURL(*dstURL, dstPipeline)
+		shareURL := fsu.NewShareURL(containerName)
+		_, err = shareURL.GetProperties(ctx)
+
+		if err == nil {
+			return err
+		}
+
+		// Create a destination share with the default service quota
```

Comment:
Do we know what the default is?  I couldn't it documented online when I had a quick look just now.

Reply:
Actually completely unsure.

Reply:
Do you have one lying around from when you tested it?  Could jump into Portal and see the size there.

Reply:
It seems as though it just defaults to 5TiB

Reply:
Perfect, that's the current maximum.  Defaulting to the max is the safest in terms of a job running successfully, so that's all good.  I'll mark this conversation resolved now.